cookbook/docs/contribute.rst

```diff
@@ -27,50 +26,56 @@ Below you’ll find instructions that will hopefully guide you through how to co
    * sandbox.config
 
    It might be easier to copy one of the existing examples and modify it to your needs.
-3. Add the example to CI
+
+3. Add the example to CI.
 
    Examples are references in `this github workflow <https://github.com/flyteorg/flytesnacks/blob/master/.github/workflows/ghcr_push.yml>`__.
    Add a new entry under ``strategy -> matrix -> directory`` with the name of your directory as well as its relative path. 
    Also, add the example to `flyte_tests_manifest.json <https://github.com/flyteorg/flytesnacks/tree/master/cookbook/flyte_tests_manifest.json>`__.
+
 4. Test your code!
-    * If the Python code can be run locally, just use ``python <my file>`` to run it.
-    * If the Python code has to be tested in a cluster:
-        * Install flytectl by running ``brew install flyteorg/homebrew-tap/flytectl``. Learn more about installation and configuration of flytectl `here <https://docs.flyte.org/projects/flytectl/en/latest/index.html>`__.
-        * Run the ``flytectl sandbox start --source=$(pwd)`` command in the directory that's one level above the directory that has Dockerfile. 
-          For example, to register `house_price_prediction <https://github.com/flyteorg/flytesnacks/tree/master/cookbook/case_studies/ml_training/house_price_prediction>`__ example, run the start command in the ``ml_training`` directory. 
-          To register ``core`` examples, run the start command in the ``cookbook`` directory. So, ``cd`` to the required directory and run all the upcoming commands in there!
-
-          Following are the commands to run if examples in ``core`` directory are to be tested on sandbox:
-            1. Build Docker container using the command: ``flytectl sandbox exec -- docker build . --tag "core:v1" -f core/Dockerfile``. 
-            2. Package the examples by running ``pyflyte --pkgs core package --image core:v1 -f``.
-            3. Register the examples by running ``flytectl register files --archive -p flytesnacks -d development --archive flyte-package.tgz --version v1``.
-            4. Visit https://localhost:30081/console to view the Flyte console, which consists of the examples present in the ``flytesnacks/cookbook/core`` directory.
-            5. To fetch new dependencies and rebuild the image, run 
-               ``flytectl sandbox exec -- docker build . --tag "core:v2" -f core/Dockerfile``, 
-               ``pyflyte --pkgs core package --image core:v2 -f``, and 
-               ``flytectl register files --archive -p flytesnacks -d development --archive flyte-package.tgz --version v2``.
-            6. Refer to `Build & Deploy Your Application “Fast”er! <https://docs.flyte.org/en/latest/getting_started_iterate.html#bonus-build-deploy-your-application-fast-er>`__ if code in itself is updated and requirements.txt is the same.
+
```

Comment:
You missed the indentation, please revert it.

Reply:
Not the whole thing, just the 'Test your code' part needs to be fixed. Even in master, I think its not indented properly.